uqcsbot/base.py

```diff
@@ -96,18 +91,19 @@ def _handle_goodbye(self, evt):
             self.logger.debug(f"Goodbye event has unexpected extras: {evt}")
         self.logger.info(f"Server is about to disconnect")
 
-    async def _handle_command(self, message: dict) -> None:
+    def _handle_command(self, message: dict) -> None:
```

Comment:
what's the difference between this `handle_command` and `run_handlers`? docs for both would be nice

Reply:
`_handle_command` does the extra execution stuff to get a command obj, whereas `_run_handlers` does it with the raw JSON message

Reply:
cool cool, docs explaining that for both would be great (Y)